### ISSUE TITLE
Cloud Sync not working on Desktop

### ISSUE THREAD

**Issue by nyxcharon** [burtonator/polar-bookshelf#1058]
**Describe the bug**
My cloud sync does not sync from my other device unless I log out and back in again.

**To Reproduce**
1. Open a document and create pagemark, comment, etc
2. Open Polar on Second Device
3. Open same document, see that it has not synced (I'ved waited up to 20 minutes after opening the app)
4. Log out of Polar
5. Log back into Polar
6. Open same document, is now synced.

**Expected behavior**
For my documents/comments/etc to be synced when I open the app. If I check the log when I open the app, I see the account login message, and then nothing until I logout/login at which point I see the sync logs

**Desktop (please complete the following information):**
 - OS: Windows 10, Ubuntu 18.04

**Additional context**
Would it be possible to get a menu item to kick off a manual sync? The logout/login is a workaround for the time being but it would be better if I could kick off a sync whenever I wanted, just in case.

**Comment by burtonator** [comment]
It should be event based... are you saying that two machines are no longer in sync when they're running. 

Once the initial/full sync is done it's just doing a delta in real time as we detect updates.

If you're on machine A + B and B isn't getting updates after A updates then it's definitely a bug.

The initial sync is mostly more like a fsck style full sync to verify we're up to date.

**Comment by nyxcharon** [comment]
Yeah, the two machines do not stay in sync unless i log out on the one that is out of date( in this example B), then back in. At that point it does a sync so that B is in sync with A. I can replicate it either way B->A or A->B they never sync until I relog

**Comment by burtonator** [comment]
Thanks.. I'll double check .. .they're all on the most recent version?

**Comment by burtonator** [comment]
Hm.. works for me.. so you have to restart polar or login/logout? 

Does the realtime sync work for a while and then break?

**Comment by nyxcharon** [comment]
Restarting polar does not help. Sync only occurs after I login/logout. 
When I first started using polar a couple weeks ago both would sync just fine.
Both clients are currently up to date.
